python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -103,8 +103,8 @@ class TableMetadata:
     column_details: list
     retention_period_in_days: int | None
     source_dataset_name: str | None = None
-    source_dataset_location: str | None = None
-    data_sensitivity_level: SecurityClassification = SecurityClassification["OFFICIAL"]
+    where_to_access_dataset: str | None = None
```

Comment:
```suggestion
    source_dataset_name: str = ""
    where_to_access_dataset: str = ""
```
type error fix from above

Reply:
Done